samples/snippets/deid.py

```diff
@@ -473,18 +473,19 @@ def reidentify_with_fpe(
 def reidentify_with_deterministic(
```

Comment:
Thanks for making the changes! We should also rename this file as reid.py (or maybe reidentify.py), since this sample demonstrates how to re-identify content. Also, a file called deid.py already exists, and it shows how to de-identify content.

Reply:
Please disregard this. I realize now that the de-identify and re-identify functions are all in the same file.